schedule-tracking/src/main/java/org/motechproject/scheduletracking/domain/json/MilestoneRecord.java

```diff
@@ -7,13 +7,21 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * <code>MilestoneRecord</code> is used to deserialize milestone from json format.
```

Comment:
it is not used to "deserialize"; it just keeps the deserialized form of an alert